datumaro/plugins/data_formats/datumaro_binary/exporter.py

```diff
@@ -75,9 +88,10 @@ def write(self):
                 self._fp = fp
                 self._sign()
                 self._dump_encryption_field()
-                self._dump_header(self.infos)
-                self._dump_header(self.categories)
-                self._dump_header(self._media_type)
+                self._dump_version()
```

Comment:
Same here. Please switch this line with `self._dump_encryption_field()`.

Reply:
Cool! I switch encryption field and version field, and unify to dump as dictionary.